docs/source/getting_started/platform.rst

```diff
@@ -15,188 +15,208 @@ This is exactly why we built the
 
-.. code-block:: python
 
-    from lightly.utils import save_embeddings
-    from lightly.api.api_workflow_client import ApiWorkflowClient
+Configuration
+^^^^^^^^^^^^^^^
 
-    # store the embeddings in a lightly compatible CSV format before uploading
-    # them to the platform
-    save_embeddings('embeddings.csv', embeddings, labels, filenames)
+In order to use the custom metadata on the Lightly Platform, it must be configured first. For this,
+follow these steps:
 
-    # upload the embeddings.csv file to the platform
-    client = ApiWorkflowClient(token='123', dataset_id='xyz')
-    client.upload_embeddings('embeddings.csv', name='my-embeddings')
+1. Go to your dataset and click on "Configurator" on the left side.
+2. Click on "Add entry" to add a new configuration.
+3. Click on "Path". Lightly should now propose different custom metadata keys.
+4. Pick the key you are interested in, set the data type, display name, and fallback value.
+5. Click on "Save changes" on the bottom.
 
+Done! You can now use the custom metadata in the "Explore" and "Analyze & Filter" screens.
 
-Sampling
+.. figure:: images/custom_metadata_weather_temperature.png
+    :align: center
+    :alt: Custom metadata weather configuration
+
+    Example of a custom metadata configuration for the key `weather.temperature`.
+
+
+Format
 ^^^^^^^^^^^
 
+In order to upload the custom metadata, you need to save it to a `.json` file in a COCO-like format.
+The following things are important: Information about the images is stored under the key `images`.
+Each image must have a `file_name` and an `id`. Custom metadata must be accessed with the `metadata` 
+key and each entry must have an `image_id` to match it with the corresponding image.
+
+For the example of an autonomous driving company mentioned above, the custom metadata file would
+need to look like this:
+
+.. code-block:: json
+
+    {
+        "images": [
+            {
+                "file_name": "image0.jpg",
+                "id": 0,
+            },
+            {
+                "file_name": "image1.jpg",
+                "id": 1,
+            }
+        ],
+        "metadata": [
+            {
+                "image_id": 0,
+                "number_of_pedestrians": 3,
+                "weather": {
+                    "scenario": "cloudy",
+                    "temperature": 20.3
+                }
+            },
+            {
+                "image_id": 1,
+                "number_of_pedestrians": 1,
+                "weather": {
+                    "scenario": "rainy",
+                    "temperature": 15.0
+                }
+            }
+        ]
+    }
+
+
+.. note:: Make sure that the custom metadata is present for every image. The metadata
+          must not necessarily include the same keys for all images but it is strongly
+          recommended.
+
+.. note:: Lightly supports integers, floats, strings, booleans, and even nested objects for
```

Comment:
dont forget arrays. Maybe we could mention somewhere that it needs to be a valid JSON object?